graphein/protein/graphs.py

```diff
@@ -382,11 +382,11 @@ def calculate_centroid_positions(
 
 def compute_edges(
     G: nx.Graph,
-    get_contacts_config: Optional[GetContactsConfig],
     funcs: List[Callable],
+    get_contacts_config: Optional[GetContactsConfig] = None,
 ) -> nx.Graph:
     """Compute edges."""
-    # Todo move to edge computation
+    # TODO: move to edge computation
```

Comment:
This is being moved inside the getcontacts-related functions on the `dssp` branch